src/main/java/seedu/foodrem/logic/commands/itemcommands/DecrementCommand.java

```diff
@@ -13,13 +13,12 @@
 import seedu.foodrem.model.Model;
 import seedu.foodrem.model.item.Item;
 import seedu.foodrem.model.item.ItemQuantity;
+import seedu.foodrem.viewmodels.ItemWithMessage;
 
 /**
  * Increments the quantity of an item by a specified amount.
  */
 public class DecrementCommand extends Command {
-    private static final String MESSAGE_SUCCESS = "Decremented Item:\n%1$s";
```

Comment:
Is there a reason why the MESSAGE_SUCCESS constant was retained in e.g. ResetCommand and HelpCommand, but removed in Decrement Command, Delete Command, Edit Command, etc...?

Reply:
Thanks for the spot! Will remove the constant from the remaining. The whole idea was the following: if
* it's only used in one place, then,
* it is clear what the purpose is for (especially if it's an argument to a method call), and so,
* abstracting it only decreases readability as checkstyle will force them to be defined all the way at the top of the class